### PR TITLE
Fix maven publishing for station simulator core

### PR DESCRIPTION
**Pull request by draganevbox**
Afer ocpp-payload package gets released the build should pass :)

### CODE REVIEW

File: simulator-core/build.gradle [hunk 1]
```diff
@@ -31,8 +31,14 @@ lombok {
...
 
+configurations {
+    internalDependencies
+}
+
+configurations.compile.extendsFrom(configurations.internalDependencies)
+
 dependencies {
-    compile project(':payload:ocpp20')
+    internalDependencies project(':payload:ocpp20')
```
Comment (RomanVlasenko) [inline simulator-core/build.gradle]: is it possible to release payload pojos as a separate artifact?

**Comment by akoshterek** [comment]
still failing tests

File: simulator-core/build.gradle (hunk 1 above)
Comment (draganevbox) [reply simulator-core/build.gradle]: yes, but then every time you change them you will have to re-release them.

I would prefer to wait until we have versioning and continous release (https://myevbox.atlassian.net/browse/SPE-833?oldIssueView=true)

**Comment by RomanVlasenko** [comment]
As far as I know all pojos are there, so we may release it only once

File: simulator-core/build.gradle (hunk 1 above)
Comment (RomanVlasenko) [reply simulator-core/build.gradle]: As far as I know all pojos are there, so we may release it only once. It can be also used on a server-side

**Pull request merged**